shennah_al-jawad/week_01/Weds-Homework.html

```diff
@@ -0,0 +1,50 @@
+<html>
+<body>
+<script>
+
+//The Word Guesser
+var word = ['D', 'O', 'G'];
+var guesses = ['_', '_', '_',];
+var counter = word.length;
+console.log(guesses);
+console.log("Let the games begin!")
+
+function guessLetter(l) {
+
+	var correctGuess = false;
+	var incorrectGuess = false;
+
+	for (var x = 0; x < word.length; x++) {
+		if (word[x] === l) {
```

Comment:
would be better to call you parameter "l" more nominating like "letter" or "guessLetter" etc